edb/schema/functions.py

```diff
@@ -1021,6 +1063,8 @@ def _create_begin(self, schema: s_schema.Schema, context):
                     f'invalid default value {p_default.text!r} of parameter '
                     f'{p.get_displayname(schema)!r}: {ex}',
                     context=self.source_context)
+            # 024
```

Comment:
What does this comment refer to?

Reply:
Sorry, this is debris (placeholders I occasionally put while analyzing the code - usually I don't forget those).